examples/custom_attributes.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2023 Alliander
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from pydantic import Field
+from pydantic.dataclasses import dataclass
+
+from pycgmes.resources.ACLineSegment import ACLineSegment
+from pycgmes.utils.profile import BaseProfile, Profile
+
+
+class CustomProfile(BaseProfile):
+    MYOWN = 100
+
+
+@dataclass
+class CustomAttribute(ACLineSegment):
```

Comment:
How can we define the namespace for this class?

Reply:
See previous comment - my understanding is that namespaces are for attributes, not classes. But a solution can be found.